app/controllers/katello/concerns/registration_commands_controller_extensions.rb

```diff
@@ -7,7 +7,7 @@ def plugin_data
         aks = ActivationKey.authorized(:view_activation_keys)
                            .where(organization_id: registration_params[:organization_id])
                            .order(:name)
-                           .map { |ak| { name: ak.name, lce: ak.environment&.name } }
+                           .map { |ak| { name: ak.name, cves: ak.content_view_environments.map(&:label).join(', ') } }
```

Comment:
Is the space after comma necessary ? does it work ?

Reply:
If you were copy/pasting to hammer with the spaces you'd need quotes. But this is just for displaying in the web UI, so I thought the spaces just looked nicer.